src/WPF/ArcGISRuntime.WPF.Viewer/Resources/SyntaxHighlighting/highlight.pack.js

```diff
@@ -1,2 +1,2 @@
 /*! highlight.js v9.12.0 | BSD3 License | git.io/hljslicense */
```

Comment:
We shouldn't make changes to the included .js files in this PR. Probably better to leave them in their original state that they were copied as.